x-pack/plugins/console_extensions/spec/overrides/data_frame.put_data_frame_transform.json

```diff
@@ -0,0 +1,25 @@
+{
+  "data_frame.put_data_frame_transform": {
+    "data_autocomplete_rules": {
+      "source": "SOURCE_INDEX_NAME",
```

Comment:
"source" and "dest" are nested like for reindex: 

```
 'source': {
        'index': '',
        'query': {
          __scope_link: 'GLOBAL.query'
        }
}
```
and

```
'dest': {
    'index': ''
}
```

same applies for the preview endpoint

Reply:
fixed